### PR TITLE
[CFM-476] notify group admins that status has changed to published.

### PR DESCRIPTION
**Pull request by anvmn**
## This PR is dependent on #1059 

https://issuetracker.amplexor.com/jira/browse/CFM-476

### CODE REVIEW

**Comment by anvmn** [comment]
selection_112

File: project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc [hunk 1]
```diff
@@ -62,9 +62,32 @@ function c4m_user_notifications_notify_user_membership_canceled($entity) {
...
+ * from 'archived' to 'published'.
+ */
+function _c4m_user_notifications_to_admin_group_status_set_published($node, $admin_id) {
+  $message = message_create('c4m_group_published', array('uid' => $admin_id));
+  c4m_message_load_tokens__c4m_group_published($message);
+
+  $wrapper = entity_metadata_wrapper('message', $message);
+  $wrapper->field_group_node->set($node->nid);
+  // Message will be saved in message_notify_send_message().
+  message_notify_send_message($wrapper->value());
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc]: `message_subscribe_send_message()`

File: project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module [hunk 2]
```diff
@@ -78,6 +79,20 @@ function _c4m_message_format_welcome_to_group_from_field(&$message, $message_ent
 /**
  * Helper function.
  *
+ * Formats 'c4m_group_published' message From field, per template requirement.
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module]: @anvmn 
Can you please write the ID of the template ?
thanks

**Comment by anvmn** [comment]
## Replaced by #1189

**Pull request closed without merging**